### ISSUE TITLE
add all BTC_*

### ISSUE THREAD

**Issue by crack00r** [miti0/mosquito#25]
may you add BTC_*
right now on all it use rly ALL currencys, but i think BTC_* ore USDT_* is enoth
i think nobody would use XMR_*

**Comment by crack00r** [comment]
BTC_* didnt work

```
root@j314793:~/mosquito# python3 mosquito.py --paper
Starting simulation..
Getting currency data: BTC_*, days left: 1
Traceback (most recent call last):
  File "mosquito.py", line 35, in <module>
    main(args)
  File "mosquito.py", line 9, in main
    engine.run()
  File "/root/mosquito/core/engine.py", line 131, in run
    self.history = self.bot.prefetch(self.strategy.get_min_history_ticks(), self.interval)
  File "/root/mosquito/core/bots/base.py", line 50, in prefetch
    backfill(args)
  File "/root/mosquito/backfill.py", line 57, in main
    candles = exchange.return_candles(pair, 300, epoch_from, epoch_to) # by default 5 minutes candles (minimum)
  File "/root/mosquito/exchanges/poloniex/polo.py", line 80, in return_candles
    return self.polo.returnChartData(currency_pair, period, start, end)
  File "/usr/local/lib/python3.5/dist-packages/poloniex/__init__.py", line 330, in returnChartData
    'end': str(end)
  File "/usr/local/lib/python3.5/dist-packages/poloniex/__init__.py", line 143, in retrying
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/poloniex/__init__.py", line 218, in __call__
    return self.handleReturned(ret.text)
  File "/usr/local/lib/python3.5/dist-packages/poloniex/__init__.py", line 270, in handleReturned
    raise PoloniexError(out['error'])
poloniex.PoloniexError: Invalid currency pair.

```

**Comment by miti0** [comment]
fixed [488d211](https://github.com/miti0/mosquito/commit/488d211c4fef8a8bd5b140b4b242239401f5447f)

Valid values:

- all - get ticker for ALL currencies
- comma separated  list of pairs, for example BTC_DGB
-  prefix_* - log all pairs with given prefix. For example BTC_* or  USDT_*
